packages/@okta/vuepress-theme-prose/assets/css/components/_top-bar.scss

```diff
@@ -1,92 +1,263 @@
 $gray-900: #2f3f4a;
-.top-bar {
-  display: block;
-  background-color: cv('brand');
-  padding: 0rem;
-  z-index: 20;
-  border-bottom: none;
 
-  &--search {
-    &--mobile {
-      @include media(">=tablet") {
-        display: none;
-      }
+.top-bar-nav{
+  display: flex;
+  flex-direction: row;
+  padding: 30px;
+  align-items: center;
+  font-size: 14px;
+  height: 95px;
```

Comment:
you might also want to adjust `.page-body`'s top padding

Reply:
It seems like there is no need in such adjustment - works fine, no noticeable difference.